src/webgl/p5.RendererGL.js

```diff
@@ -1920,6 +2030,9 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
     fillShader.setUniform('uEmissive', this._useEmissiveMaterial);
     fillShader.setUniform('uShininess', this._useShininess);
 
+    // calling the _setImageLightUniforms from here
```

Comment:
This one probably doesn't need a comment, since it doesn't add much over the code itself